src/Modal/_ModalDialog.scss

```
@@ -38,6 +38,14 @@
       top: calc(#{$modal-inner-padding} / 2 * -1);
     }
   }
+
+  &.pgn__modal-visible-overflow {
+    overflow: visible;
+
+    .pgn__modal-body {
+      overflow: visible;
+    }
+  }
```

Please try this. I think this option looks better:

```suggestion
  &.pgn__modal-visible-overflow .pgn__modal-body {
    overflow: visible;
  }
```

---

That doesn't work as we need to overflow both `pgn__modal-body` and `pgn__modal`. Your suggestion is missing the
```scss
  &.pgn__modal-visible-overflow {
    overflow: visible;
  }
```
part.